### DISCUSSION TITLE
Why is footer using `grid`?

### DISCUSSION THREAD

**Discussion by StevenTCramer** [microsoft/fluentui-blazor#1777] [Q&A]
```
.footer {
    display: grid;
    z-index: 10;
    flex-direction: column;
    position: relative;
    font-family: var(--body-font);
    font-weight: normal;
    font-size: var(--type-ramp-minus-1-font-size);
    line-height: var(--type-ramp-minus-1-line-height);
}
```

If setting `display` to `grid` then `flex-direction` doesn't make sense.

I would prefer it be flex but I don't know the reasoning for the decisions.

If I want to change it to flex in localized css how can I do that?

```
 ::deep > footer {
  display: flex;
  flex-direction: row;
  padding: 10px 10px;
}
```
The above gets applied but is always overridden by the Fluent CSS which puts it back to `grid`.

The only way I have figured out how to change it is with `Style="display: flex; flex-direction: row;"`

**Comment by StevenTCramer** [comment]
![image](https://github.com/microsoft/fluentui-blazor/assets/357219/130861ab-1dea-44c6-99ed-362f30afb835)
Here one can see the Styles. The Style tag > FluentUI lib > isolated css > site.css > app.css

**Comment by vnbaaij** [comment]
I think this just comes from a bad design decission in the past. In the demo site we are overwriting it as well:
![image](https://github.com/microsoft/fluentui-blazor/assets/1761079/96888ecd-b40e-4b54-b9c1-ade9e86d6eb2)

Noticed it before, but did not change it then because it might be considered a breaking change (albeit a small one for sure) but will change it for the next release to:
```
.footer {
    display: flex;
    z-index: 10;
    flex-direction: row;
    font-family: var(--body-font);
    font-weight: normal;
    font-size: var(--type-ramp-minus-1-font-size);
    line-height: var(--type-ramp-minus-1-line-height);
}
```

For the templates this means that I'll add a `FluentSpacer` to the footer so the texts will still align left an right. I'll also remove some then redundant CSS for those links from the `app.css`

**Reply by vnbaaij** [reply]
PR is created for this: #1778
